sql/slave.cc

```diff
@@ -3117,6 +3117,10 @@ void show_master_info_get_fields(THD *thd, List<Item> *field_list,
                           Item_return_int(thd, "Executed_log_entries", 10,
                                           MYSQL_TYPE_LONG),
                           mem_root);
+    field_list->push_back(new (mem_root)
```

Comment:
As `slave_max_statement_time` option is supposed to be pretty marginal in the usage terms, I'd like
more to have its SSS record to be "also" optional :-). That is why not to prepare with
```
if (slave_max_statement_time > 0) ... 
``` (and the same (to store the result of the check in a temp var of course) check on l.3372?

Reply:
Agreed, can do.